src/platforms/python/guides/celery/index.mdx

```diff
@@ -8,34 +8,34 @@ description: "Learn about using Sentry with Celery."
 
 The Celery integration adds support for the [Celery Task Queue System](https://docs.celeryq.dev/).
 
-Just add `CeleryIntegration()` to your `integrations` list:
+## Install
+
+Install `sentry-sdk` from PyPI with the `celery` extra:
+
+```bash
+pip install --upgrade 'sentry-sdk[celery]'
+```
+
+## Configure
+
+If you have the `celery` package in your dependencies, the Celery integration will be enabled automatically when you initialize the Sentry SDK.
+
+Generally, make sure that the **call to `init` is loaded on worker startup**, and not only in the module where your tasks are defined. Otherwise, the initialization happens too late and events might end up not being reported.
```

Comment:
```suggestion
Generally, make sure that the **call to `init` is loaded on worker startup**, and not only in the module where your tasks are defined. Otherwise, the initialization happens too late and events might end up not being reported.
```
```suggestion
Make sure that the **call to `init` is loaded on worker startup**, and not only in the module where your tasks are defined. Otherwise, the initialization happens too late and events might end up not being reported.
```